cmd/agent/app/agent.go

```diff
@@ -53,8 +53,10 @@ func NewAgentCommand(ctx context.Context) *cobra.Command {
 	karmadaConfig := karmadactl.NewKarmadaConfig(clientcmd.NewDefaultPathOptions())
 
 	cmd := &cobra.Command{
-		Use:  "karmada-agent",
-		Long: `The karmada agent runs the cluster registration agent`,
+		Use: "karmada-agent",
+		Long: `The karmada-agent runs the cluster registration agent on the member cluster side. 
+It will pull the user-submitted application and run it locally. Examples of controllers in karmada-agent
+are cluster-status-controller, execution-controller, work-status-controller and service-export-controller.`,
```

Comment:
Is the controller name the same as that here:

https://github.com/karmada-io/karmada/blob/master/cmd/agent/app/agent.go#L102-L105

Reply:
Yes, I am inspired from that. I wonder if the above description accurate for some components.

Reply:
It's a clear read for me.

Reply:
What means example? means this agent has `cluster-status-controller, execution-controller, work-status-controller and service-export-controller`? If so, maybe describe like `it contains xxx`?

Reply:
We don't restrict `karmada-agent` running `on` member clusters. It can be outside of a cluster.